app/routes/docs.vtp.mdx

```diff
@@ -0,0 +1,226 @@
+
+## The Future of VTP
+
+IVOA and other organizations including the new GCN, Rubin, SCiMMA, have moved away from VTP in favor of more modern, general-purpose data distribution protocols like [Kafka](client). Kafka is widely used throughout industry, is well supported, and is growing in its use in astronomy. As GCN services move from GCN Classic to the new GCN, we are planning to retire the existing GCN Classic VOEvent system as it is currently implemented in several cloud-based servers that do not meet NASA standards. The GCN team is gradually migrating to or replacing GCN Classic services in the new GCN, and is not planning any new development in GCN Classic.
+
+We intend for this transition to be planned, announced, and implemented with ample warning for the user community. We propose 3 alternatives for how to complete this transition, and are soliciting [user feedback](vtp#feedback-survey). We know that many pipelines, robotic telescopes, and users depend on this service and we aim to be minimally distriptive to their operations.
+
+## GCN VTP Migration Options
+
+The GCN team is exploring the following options to meet the VOEvent needs of the GCN user community. The current GCN VOEvent broker must be updated to meet NASA standards, with a minimum change for VOEvent users to update their server names.
+
+<br />
+<CardGroup>
+  <Card gridLayout={{ tablet: { col: 4 } }} headerFirst>
+    <CardHeader className="bg-base-lighter">
+      <p className="margin-0 text-base-dark">Option 1:</p>
+      <h4 className="usa-card__heading">
+        Replace VTP with VOEvents over Kafka
+      </h4>
+    </CardHeader>
+    <CardBody>
+      <IconList>
+        <IconListItem>
+          <IconListIcon className="text-green">
+            <Icon.Check aria-hidden="true" />
+          </IconListIcon>
+          <IconListContent>
+            Service already exists and is available for use
+          </IconListContent>
+        </IconListItem>
+        <IconListItem>
+          <IconListIcon className="text-green">
+            <Icon.Check aria-hidden="true" />
+          </IconListIcon>
+          <IconListContent>Self service subscription</IconListContent>
+        </IconListItem>
+        <IconListItem>
+          <IconListIcon className="text-green">
+            <Icon.Check aria-hidden="true" />
+          </IconListIcon>
+          <IconListContent>Open source</IconListContent>
+        </IconListItem>
+        <IconListItem>
+          <IconListIcon className="text-green">
+            <Icon.Check aria-hidden="true" />
+          </IconListIcon>
+          <IconListContent>Transport security layer</IconListContent>
+        </IconListItem>
+        <IconListItem>
+          <IconListIcon className="text-green">
+            <Icon.Check aria-hidden="true" />
+          </IconListIcon>
+          <IconListContent>
+            Broker is client, minimal firewall configuration{' '}
```

Comment:
No, the broker is the server. The producers and consumers are clients. Same for Comet.